oaknational/Oak-Web-Application
# fix(dissmissable-card): title

## Description

- Small change to DismissableCard  so that it shows the correct a11y title on the close button (and the console warning goes away)
- Adds test for DismissibleCard

## Issue(s)

Fixes #389 

## How to test

1. Go to {[cloud link](https://oak-web-application-efx1h9ibn-oak-national-academy.vercel.app/)}
2. Hover over the close button on the Oak Beta card
3. You should see text "Dismiss Oak beta banner" 

## Screenshots

How it used to look (delete if n/a):

![image](https://user-images.githubusercontent.com/12934669/181258878-3bd2c85d-9fa3-4654-b9c8-0a7342280032.png)


How it should now look:

![image](https://user-images.githubusercontent.com/12934669/181258757-fdb57c5f-4dc4-479e-b84e-0cb5bc1295c0.png)


## Checklist
